indico/modules/events/editing/controllers/backend.py

```diff
@@ -250,3 +259,27 @@ def _process_PATCH(self, text, internal):
     def _process_DELETE(self):
         delete_revision_comment(self.comment)
         return '', 204
+
+
+class RHRevisionFilesExportZIP(RHContributionEditableRevisionBase):
+    def _check_revision_access(self):
+        return self._user_is_authorized_submitter() or self._user_is_authorized_editor()
+
+    def _process(self):
+        if not self.revision.files:
+            raise NotFound
```

Comment:
When would you have a revision without files? Anyway, not sure if we even need this check, you'd just get an empty zip file in this case.